api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -482,38 +482,68 @@ private void setupDefaultEncounterTemplates() {
     // TODO does this need to be modified for the new visit note at all?
     private void enableVisitManagement() {
 
-        extensions.add(overallAction(Extensions.CREATE_VISIT_OVERALL_ACTION,
-                "coreapps.task.startVisit.label",
-                "fas fa-fw icon-check-in",
-                "script",
-                "visit.showQuickVisitCreationDialog({{patient.patientId}})",
-                "Task: coreapps.createVisit",
-                and(patientDoesNotActiveVisit(), patientNotDead())));
+        if (config.getCountry().equals(ConfigDescriptor.Country.PERU)) {
```

Comment:
Instead of duplicating all this code, why don't you just say "if the country is NOT Peru, add the merge visits action"?